client/blocks/reader-post-card/byline.jsx

```diff
@@ -79,7 +80,7 @@ class PostByline extends React.Component {
 							author={ post.author }
 							siteUrl={ streamUrl }
 							post={ post }>
-							{ post.author.name }
+							{ decodeEntities( post.author.name ) }
```

Comment:
r152765-wpcom decodes author names. updated the linked feed to pick up the new author names.